2371/2/author_entity.rb

```diff
@@ -0,0 +1,43 @@
+require File.expand_path(File.dirname(__FILE__) + '/constants')
+require File.expand_path(File.dirname(__FILE__) + '/battle_entity')
+
+# AuthorEntity responsible for author info
+class AuthorEntity
```

Comment:
You could just name it as `Author`. No need to add Entity word.
OOP is about business entities. When you create `Author` class it's clear that this class will represent an author.